src/main/java/org/jenkinsci/maven/plugins/hpi/AbstractJenkinsManifestMojo.java

```diff
@@ -266,4 +275,39 @@ protected Manifest loadManifest(File f) throws IOException, ManifestException {
             IOUtil.close(r);
         }
     }
+
+    private void addLicenseAttributesForManifest(Manifest.Section target) throws ManifestException {
+        final List<License> licenses = project.getLicenses();
+        int licenseCounter = 1;
+        for (License lic : licenses) {
+            addAttributeIfNotNull(target, "Plugin-License-Name-" + licenseCounter, lic.getName());
+            addAttributeIfNotNull(target, "Plugin-License-Url-" + licenseCounter, lic.getUrl());
+            addAttributeIfNotNull(target, "Plugin-License-Distribution-" + licenseCounter, lic.getDistribution());
+            addAttributeIfNotNull(target, "Plugin-License-Comments-" + licenseCounter, lic.getComments());
```

Comment:
Comments? Really? Let us only support the attributes that we have actual, proven use cases for in downstream PRs.